app/client/ui/WidgetTitle.ts

```diff
@@ -144,30 +222,41 @@ function buildWidgetRenamePopup(ctrl: IOpenController, vs: ViewSectionRec, optio
         inputTableName,
         updateOnKey,
         {disabled: isSummary, placeholder: t("Provide a table name")},
-        testId('table-name-input')
+        testId('table-name-input'),
+        commandGroup.attach(),
       ),
     ]),
     dom.maybe(!options.widgetNameHidden, () => [
       cssLabel(t("WIDGET TITLE")),
       widgetInput = cssInput(inputWidgetTitle, updateOnKey, {placeholder: inputWidgetPlaceholder},
-        testId('section-name-input')
+        testId('section-name-input'),
+        commandGroup.attach(),
       ),
     ]),
+    cssLabel(t("WIDGET DESCRIPTION")),
+    descInput = cssTextArea(inputWidgetDesc, updateOnKey,
```

Comment:
What do you think about moving the `cssTextArea` out of `ColumnTitle.ts` and into a new file, `DescriptionTextArea.ts`, and exporting a function from it that creates it, taking optional dom arguments as a parameter?

Both `WidgetTitle.ts` and `ColumnTitle.ts` could then use it, passing in their own command groups and test ids.

Reply:
Maybe we can create file with all common styles as RightPanelStyles, because it's the main part to factorize